src/main/java/com/linecorp/armeria/common/metrics/DropwizardMetricConsumer.java

```diff
@@ -33,8 +33,16 @@ public DropwizardMetricConsumer(MetricRegistry metricRegistry, String metricName
     }
 
     @Override
+    public void invocationStart(Scheme scheme, String hostname, String path, Optional<String> method) {
+        final String metricName = MetricRegistry.name(metricNamePrefix, method.orElse("__unknown__"));
+        final DropwizardRequestMetrics metrics = getRequestMetrics(metricName);
+        metrics.markStart();
+    }
+
+    @Override
     public void invocationComplete(Scheme scheme, int code, long processTimeNanos, int requestSize,
```

Comment:
That being said, could we count by code instead of success/failure? e.g. create the meters bound at `.../code/<codeIntVal>` instead of `.../successes` and `.../failures`?